packages/sdk/src/Orders.ts

```diff
@@ -6,25 +8,28 @@ export class Orders {
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#create-order
 	 */
-	public create(order: $FixMe) {
-		return this.client.post('/orders', order)
+	public create(order: T.OrdersCreate) {
+		return this.client.post<T.OrdersCreateResponse>('/orders', order)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#get-order
 	 */
-	public get(orderId: $FixMe) {
-		return this.client.get(`/orders/${encode(orderId)}`)
+	public get(orderId: string) {
+		return this.client.get<T.OrdersGetResponse>(`/orders/${encode(orderId)}`)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#update-order
 	 */
-	public update(order: $FixMe) {
-		return this.client.put(`/orders/${encode(order.id || order.source_id)}`, omit(order, ['id']))
+	public update(order: T.OrdersUpdate) {
+		return this.client.put<T.OrdersUpdateResponse>(
+			`/orders/${encode(order.id || order.source_id)}`,
+			omit(order, ['id']),
+		)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#list-orders
 	 */
-	public list(params: $FixMe) {
-		return this.client.get('/orders', params)
+	public list(params: T.OrdersListParams) {
```

Comment:
It should allow to be called without any parameter at all and just default to `{}`